src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,17 +1,28 @@
 package edu.kis.vh.nursery;
 
-public class HanoiRhymer extends defaultCountingOutRhymer {
+public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-int totalRejected = 0;
+	private int totalRejected = 0;
 
 	public int reportRejected() {
-		return totalRejected;
+		return getTotalRejected();
 	}
 
 	public void countIn(int in) {
 	if (!callCheck() && in > peekaboo())
-			totalRejected++;
-			else
-				super.countIn(in);
+		setTotalRejected(getTotalRejected() + 1);
```

Comment:
Using getter/setter to access field is usually an exaggeration (keep field reference)